datavault-webapp/src/main/java/org/datavaultplatform/webapp/controllers/RoleAssignmentController.java

```diff
@@ -40,7 +40,7 @@ public ResponseEntity updateRoleAssignment(
             @PathVariable("roleType") String type,
             @PathVariable("target") String targetId,
             @Valid @NotNull @RequestParam("assignment") Long assignmentId,
-            @Valid @NotNull(message = "Must select a role") @RequestParam("role") Long roleId) {
+            @Valid @NotNull(message = "Pleease specify a role") @RequestParam("role") Long roleId) {
```

Comment:
Pretty Pleeeease?

Reply:
"There's no point in managing these labels via resource bundles", they said